README.md

```diff
@@ -42,114 +85,15 @@ Install this package using the [Swift Package Manager](https://www.swift.org/doc
 - **Have a bug to report?** [Open an issue](https://github.com/maplibre/maplibre-navigation-ios/issues). If possible, include the version of Maplibre Services, a full log, and a project that shows the issue.
 - **Have a feature request?** [Open an issue](https://github.com/maplibre/maplibre-navigation-ios/issues/new). Tell us what the feature should do and why you want the feature.
 
-## <a name="sample-code">Sample code
+## Sample code
 
-A demo app is currently not available. Please check the Mapbox repository or documentation for examples, especially on the forked version. You can try the provided demo app, which you need to first run `carthage update --platform iOS --use-xcframeworks` for in the root of this project.
+We do provide a limited example app but its not functional right out of the box. 
```

Comment:
🎉

Reply:
This is great!

Maybe you could elaborate slightly on what exactly it does / does not do in the README.